src/oc_erchef/apps/oc_chef_authz/src/oc_chef_authz_db.erl

```diff
@@ -182,3 +183,18 @@ fetch_group(#oc_chef_authz_context{reqid = ReqId, server_api_version = ApiVersio
         {error, _} = Error ->
             Error
     end.
+
+find_org_actors_by_name(OrgId, ActorNames) ->
+    case sqerl:select(find_org_actors_by_name, [OrgId, ActorNames]) of
+        {ok, L} when is_list(L) ->
+            % WIP - change this to a record.
```

Comment:
We are bad at coming back to WIP and TODO comments.  Let's remove this unless we think we are gonna do it in a soon-to-come follow up PR.